src/predicates.ts

```diff
@@ -894,11 +894,11 @@ export class ContainerVolumesFromMissing implements Predicate<any> {
       return { matched: false };
 
     }
-    const matches: RuleMatchedAt[] = _.flatMap(root.components, (component, componentIndex) => {
+    const matches: RuleMatchedAt[] = _.flatMap(root.components, (component: Component, componentIndex) => {
       if (_.isEmpty(component.containers)) {
         return [{ matched: false }];
       }
-      return _.flatMap(component.containers, (container, containerIndex) => {
```

Comment:
TS errors here and at 1102 - should probably be fixed before merge

Reply:
yeah there seem to be some new type errors all of a  sudden. Will take another pass.